memcached/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM debian:jessie
```

Comment:
https://github.com/autopilotpattern/memcached/pull/2 looks good. See my comments in Basecamp and the PR, and merge!